src/app/support/donationBox.tsx

```diff
@@ -121,9 +223,43 @@ function DonationBox() {
     console.log("Form Errors:", errors);
   };
 
+  // donation emails
+  const sendFellowDonationEmail = async (
+    email: string,
+    name: string,
+    amount: string,
+  ) => {
+    const firstName = name.split(" ")[0];
+    console.log(firstName);
+    await fetch("/api/emails/donationEmails/fellowDonationEmail", {
+      method: "POST",
+      body: JSON.stringify({
+        email: email,
+        firstName: firstName,
+        amount: amount,
+      }),
+    });
+  };
+
+  const sendBusinessDonationEmail = async (
+    email: string,
+    businessName: string,
+    amount: string,
+  ) => {
+    await fetch("/api/emails/donationEmails/businessDonationEmail", {
+      method: "POST",
+      body: JSON.stringify({
+        email: email,
+        businessName: businessName,
+        amount: amount,
+      }),
+    });
+  };
+
   //setting chosen amount for donation via buttons
   const handleAmountChange = (amount: any) => {
     setCustomAmount("");
+
     //remove amounts if a button is already selected then gets clicked again
     if (String(amount) === selectedAmount) {
```

Comment:
Since this check seems to be used over and over, you can create a function to set the values instead. It can take the amount as an input "0" or amount.
```suggestion
    function setDollarAmount(amount) {
        if (donationCategory === "individual") {
        setIndividualValue("amount", amount);
      } else {
        setBusinessValue("amount", amount);
      }
    }
```

Reply:
Then use that function here and below by just calling setDollarAmount(amount)